cmd/podman/spec.go

```diff
@@ -425,28 +423,28 @@ func (c *createConfig) CreateBlockIO() (*spec.LinuxBlockIO, error) {
 		bio.WeightDevice = lwds
 	}
 	if len(c.Resources.DeviceReadBps) > 0 {
-		readBps, err := makeThrottleArray(c.Resources.DeviceReadBps)
+		readBps, err := makeThrottleArray(c.Resources.DeviceReadBps, "bps")
```

Comment:
Can you make "bps" and "iops" into enum constants, and just pass them in.

Reply:
Done